src/js/lib/tracking.js

```diff
@@ -1,34 +1,78 @@
 import events from 'events';
 
 class Tracker {
-    static getEventObject(videoId, event) {
+    buildOphanEventObject(event) {
         return {
             video: {
-                id: `gu-video-youtube-${videoId}`,
+                id: this.trackingId,
                 eventType: `video:content:${event}`
             }
         };
     }
 
-    track (property) {
+    buildGoogleAnalyticsEventObject(event) {
+        // copy `baseEventObject`
+        const eventObject = Object.assign({}, this.googleAnalyticsProperties.baseEventObject);
+        eventObject[this.googleAnalyticsProperties.metricMap[event]] = 1;
+        return eventObject;
+    }
+
+    track(property) {
         this.emitter.emit(property);
     }
 
     constructor(properties) {
-        this.videoId = properties.videoId;
+        const self = this;
+
+        self.videoId = properties.videoId;
+
+        self.trackingId = `gu-video-youtube-${this.videoId}`;
```

Comment:
made up - it follows the pattern used in [frontend](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/hosted/youtube.js#L36)